internal/backend/remote-state/inmem/backend.go

```diff
@@ -48,19 +50,57 @@ func Reset() {
 
 // New creates a new backend for Inmem remote state.
 func New() backend.Backend {
+	if os.Getenv("TF_INMEM_TEST") != "" {
+		// We use a different schema for testing. This isn't user facing unless they
+		// dig into the code.
+		fmt.Sprintln("TF_INMEM_TEST is set: Using test schema for the inmem backend")
```

Comment:
This is using `Sprintln`, which returns a string, as opposed to printing to the CLI. If that was intentional, then this can simply be a comment.